katello/hooks/pre_commit/13-hiera.rb

```diff
@@ -1,16 +1,43 @@
-TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
+# A mapping of tuning profile to the required CPU cores and memory in GB
+# TODO: check actual required values
+TUNING_SIZES = {
+  'default' => [1, 8],
+  'medium' => [1, 10],
+  'large' => [1, 20],
+  'extra-large' => [1, 128],
+  'extra-extra-large' => [1, 256],
+}
 TUNING_FACT = 'tuning'
 
 current = get_custom_fact(TUNING_FACT)
 new = app_value(:tuning)
```

Comment:
`new` is a keyword, can we use something else?

Reply:
Already existed, but good point.